src/Controller/PayPalButtonsController.php

```diff
@@ -135,4 +150,39 @@ public function renderPaymentPageButtonsAction(Request $request): Response
             return new Response('');
         }
     }
+
+    public function renderPayPalPaymentAction(Request $request): Response
```

Comment:
I'm not a huge fan of the "no verification at all" to who is the owner of the order. Changing the `orderId` in the request could lead to some issues here. Is it right?